coq/coq.el

```diff
@@ -115,6 +115,18 @@ Namely, goals that do not fit in the goals window."
   :type 'boolean
   :group 'coq)
 
+(defcustom coq-goal-conclusion-marker "========"
+  "Marks the part of interest in each goal to focus the goal buffer on"
+  :type 'regexp
+  :group 'coq
+  )
+
+(defcustom coq-center-goal-after-focus 'top
+  "Where to display the goal marker after rendering a proof goal"
+  :type '(radio
+    (const :tag "move conclusion marker to the top of the window" top)
+    (const :tag "move conclusion marker to the middle of the window" middle)
+    (const :tag "move conclusion marker to the bottom of the window" bottom)))
 
```

Comment:
For better user configuration should we merge this preference with `coq-prefer-top-of-conclusion`?
Currently if this one is nil then we  show the end of the conclusion (of the first goal). Something like:

```
    (const :tag "focus on the end of the conclusion even if it hides the conclusion marker" tail)
```

`coq-prefer-top-of-conclusion`could be left for compatibilty.
@cipher1024 what do you think?